INSTALLATION.md

```diff
@@ -0,0 +1,91 @@
+# Hugo installation and Setup
+
+## Windows (CLI Users)
+
+1) Visit the link [Hugo Releases](https://github.com/gohugoio/hugo/releases) and download the latest release of hugo for Windows.(Hugo 0.53 at the time of writing this).
```

Comment:
```suggestion
1) Visit the link [Hugo Releases](https://github.com/gohugoio/hugo/releases) and download the latest release of hugo for Windows.(Hugo 0.55.6 at the time of writing this).
```
Might as well

Reply:
Will remove that, why keep things that are dynamic and out of our control..